net/nft-qos/files/nft-qos.config

```diff
@@ -51,6 +51,9 @@ config default default
 	# list limit_whitelist '192.168.1.0/24'
 	# list limit_whitelist 'ABCD:CDEF::1/64'
 
+  # Option for Mac address based traffic control
+  option limit_mac_enable '0'
```

Comment:
Sorry again
Please also fix space vs tabs

Reply:
Thanks. Fixed in the new commit.